ext/PkgREPLMode/PkgREPLMode.jl

```diff
@@ -1,16 +1,22 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
-module REPLMode
+module PkgREPLMode
```

Comment:
```suggestion
module REPLExt
```
I really don't like the trend to put the parent package name first and this should bear the Ext name

Reply:
How about `REPLModeExt`?